presto-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -204,6 +204,7 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
 
     componentDidMount() {
         this.refreshLoop.bind(this)();
+        new window.ClipboardJS('.copy-button');
```

Comment:
technically, the `window.` is redundant. What's the rationale?

Reply:
@flow checker

Reply:
so it's a "best practise". We don't use it on CI, do we?

Reply:
We do (on couple of files) :)

Here: https://github.com/prestosql/presto/blob/master/presto-main/src/main/resources/webapp/src/components/LivePlan.jsx#L14,
https://github.com/prestosql/presto/blob/master/presto-main/src/main/resources/webapp/src/components/PageTitle.jsx#L14

Reply:
Flow is run here: https://github.com/prestosql/presto/blob/master/presto-main/bin/check_webui.sh#L40